Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -48,28 +48,87 @@ namespace Hazel {
 			s_GLFWInitialized = true;
 		}
 
-		m_Window = glfwCreateWindow((int)props.Width, (int)props.Height, m_Data.Title.c_str(), nullptr, nullptr);
+		GLFWmonitor* windowRenderScreen = glfwGetPrimaryMonitor(); // where the window will be rendered
+		const GLFWvidmode* baseVideoMode = glfwGetVideoMode(windowRenderScreen);
+
+		// Set the actual width and height
+		switch (m_Data.Mode)
+		{
+			case WindowMode::Borderless:
+			case WindowMode::Fullscreen:
+				m_Data.Width = baseVideoMode->width;
+				m_Data.Height = baseVideoMode->height;
+				break;
+			case WindowMode::Windowed:
+				m_Data.Width = props.Width;
+				m_Data.Height = props.Height;
+				break;
+		}
+
+		HZ_CORE_INFO("Creating window '{0}' ({1}, {2})", m_Data.Title, m_Data.Width, m_Data.Height);
 
+		// Create window in windowed mode
```

Comment:
This is an old comment, this should be removed as it is incorrect: we are creating the window in the correct mode directly.